sources/packages/backend/libs/test-utils/src/factories/program-year.ts

```diff
@@ -19,3 +26,26 @@ export function createFakeProgramYear(programYearPrefix?: number): ProgramYear {
   programYear.endDate = `${endingYear}-07-31`;
   return programYear;
 }
+
+/**
+ * Ensure that a program year exists, checking by its name.
+ * If it does not exists create a default using {@link createFakeProgramYear}
+ * and save it.
+ * @param db e2e data sources.
+ * @param programYearPrefix program year prefix, for instance, a prefix 2000
+ * would create a program year 2000-2001.
+ * @returns program year with the prefix.
+ */
+export async function ensureProgramYearExists(
```

Comment:
how about  the method name `getOrCreateProgramYear`

Reply:
Either way would work for me. Lest see if there is any input from other devs and please let me know if it would be a blocker 😉

Reply:
no, its a suggestion